### ISSUE TITLE
[PyPi] Deprecation warning

### ISSUE THREAD

**Issue by julesbouton** [XachaB/Qumin#33]
When installing Qumin with pypi, I get the following warning:

```
DEPRECATION: qumin is being installed using the legacy 'setup.py install' method, 
because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. 
pip 23.1 will enforce this behaviour change. A possible replacement is to enable 
the '--use-pep517' option. 
Discussion can be found at https://github.com/pypa/pip/issues/8559
```

If I understand it well, we should use a `pyproject.toml` file instead of `setup.py` ? Don't know it this is project related or only wrong setups on my side. Do not hesitate to close if irrelevant.

**Comment by XachaB** [comment]
Hi Jules, I think if you just install `wheel`, the warning will disappear.

But it is correct that it would be better for us to migrate from `setup.cfg` to `pyproject.toml`, so I'll keep the issue :)

**Comment by XachaB** [comment]
No, I think you need wheel first, in your environment, in order to install the Qumin with wheel (not as a dependency of Qumin). In the same way that you don't add "pip" as a requirement, I think.

I'm also with a low connection (currently in a bus, later in several trains), but having a look at your first MR :)

**Comment by XachaB** [comment]
You're right, I get this too.

I think we need to both migrate to `pyproject.toml` and for me to learn again how to distribute projects with this setup.

**Comment by XachaB** [comment]
Migrated: https://gitlab.com/qumin/Qumin/-/issues/33
